source/sio/io_uring/static_io_pool.cpp

```diff
@@ -10,10 +10,11 @@ namespace sio::io_uring {
     context.run_until_stopped();
   }
 
-  static_io_pool::static_io_pool(std::size_t nthreads) {
-    contexts_.resize(nthreads);
+  static_io_pool::static_io_pool(std::size_t nthreads, unsigned iodepth) {
+    iodepth = (iodepth + nthreads - 1) / nthreads;
```

Comment:
I would rather prefer to specify the same iodepth for each worker thread. I.e. just forward this argument to all the constructors